src/alfasim_score/converter/alfacase/convert_alfacase.py

```diff
@@ -0,0 +1,49 @@
+from alfasim_sdk import AnnulusDescription
+from alfasim_sdk import FormationDescription
+from alfasim_sdk import ProfileDescription
+from alfasim_sdk import WellDescription
+from alfasim_sdk import XAndYDescription
+from barril.units import Scalar
+
+from alfasim_score.constants import ANNULUS_TOP_NODE_NAME
+from alfasim_score.constants import WELLBORE_BOTTOM_NODE
+from alfasim_score.constants import WELLBORE_TOP_NODE
+from alfasim_score.converter.alfacase.score_input_reader import ScoreInputReader
+from alfasim_score.units import LENGTH_UNIT
+
+
+class AlfacaseConverter:
+    def __init__(self, score_reader: ScoreInputReader):
+        self.score_input = score_reader
+        self.well_name = score_reader.score_filepath.stem
```

Comment:
Get `name` from json file. In our example, it is generic, however it should have the name of the well along side some study tag. Thus, I believe it is a good field for it.

Reply:
Good point! I'm gonna change this